octobot_trading/constants.py

```diff
@@ -42,6 +42,9 @@
 DEFAULT_BACKTESTING_TIME_LAG = 0
 INFINITE_MAX_HANDLED_PAIRS_WITH_TIMEFRAME = -1
 
+FULL_CANDLE_HISTORY_EXCHANGES = ["bequant", "binance", "binanceus", "binanceusdm", "bitcoincom",
+                            "bitfinex", "bitfinex2", "bitmex", "idex"]
```

Comment:
kucoin is also used a lot, is it compatible ?

Reply:
To ba available the exchange have to supprt argument since=0 but kucoin return nothing
`get_symbol_prices(symbol, time_frame, since=0)`